src/components/goodList/goodList.jsx

```diff
@@ -0,0 +1,13 @@
+import { GoodCard } from '../goodCard/GoodCard';
+
+export const GoodList = ({ goods }) => {
+  return (
+    <section className="TodoList">
+      <ul>
+        {goods.map(good => (
+          <GoodCard good={good} />
```

Comment:
Don't forget about the key attribute
```suggestion
          <GoodCard good={good} key={good} />
```